Task on exceptions on python
